### ISSUE TITLE
Native symbols upload does not work when using custom buildDir

### ISSUE THREAD

**Issue by janicduplessis** [getsentry/sentry-android-gradle-plugin#157]
_Android Gradle Plugin:_
**4.2.1**

_Gradle:_
**6.9.0**

_Optimizer:_
- [ ]  ProGuard
- [x]  R8

_Have you created the sentry.properties file?_
- [ ]  Yes -> where?
- [ ]  No

_SDKs:_
- [ ]  sentry-android -> which version?
- [x]  sentry-react-native -> 2.6.1

---
I have the following issue:

Native symbol upload doesn't work when using a different build folder. I think [this](https://github.com/getsentry/sentry-android-gradle-plugin/blob/main/plugin-build/src/main/kotlin/io/sentry/android/gradle/tasks/SentryUploadNativeSymbolsTask.kt#L80) should use project.buildDir instead and should fix the issue, but not sure how to test this change locally.

**Steps to reproduce:**

In root build.gradle add

```groovy
allprojects {
    buildDir = System.getProperty("user.home")}/.gradle/builds/${rootProject.name}/${project.name}"
}
```

This will build the project in ~/.gradle/builds instead of the build directory in each project folder.

**Actual result:**

```
> Task :app:uploadSentryNativeSymbolsForRelease
> Found 0 debug information files
> No debug information files found
```

**Expected result:**
Native symbols that exist in the merged_native_libs folder get uploaded.

**Comment by janicduplessis** [comment]
I can confirm that removing the buildDir config fixes the issue.

**Comment by cortinico** [comment]
Yup that's happening sadly because the `build/` path is hardcoded inside the plugin. This will fix it #158
